src/FSharp.Analyzers/PartialAppAnalyzer.fs

```diff
@@ -9,7 +9,15 @@ open Microsoft.FSharp.Core
 [<Literal>]
 let Code = "GRA-PARTAPP-001"
 
-type Handler = string * FSharp.Compiler.Text.range * int -> unit
+type AppHandler = string * FSharp.Compiler.Text.range * int -> unit
+
+type PipeHandler = FSharp.Compiler.Text.range * int -> unit
+
+type Handlers =
+    {
+        AppHandler : AppHandler
+        PipeHandler : PipeHandler
+    }
```

Comment:
I think there is an issue in `collectExprFromMatchClauses`.

`resultExpr` should be de-indented on one level.

Reply:
```fsharp
let collectExprFromMatchClauses (clauses : SynMatchClause list) =
    clauses
    |> List.collect (fun (SynMatchClause (whenExpr = whenExpr ; resultExpr = resultExpr)) ->
        [
            match whenExpr with
            | None -> ()
            | Some whenExpr -> yield whenExpr
            yield resultExpr
        ]
    )
```